lib/firmata.js

```diff
@@ -545,6 +545,17 @@ var Board = function(port, options, callback) {
         this.setSamplingInterval(settings.samplingInterval);
       }
       if (settings.skipCapabilities) {
```

Comment:
however I have not thought this through regarding the overall impact to the existing code. Would need to think more about it and discuss with @rwaldron